Extensions/timestamps.js

```diff
@@ -68,15 +68,54 @@ XKit.extensions.timestamps = new Object({
 					XKit.storage.set("timestamps", `extension__setting__${x}`, this.preferences[x].value.toString());
 				}
 			}
+			XKit.storage.set("timestamps", "preference_conversion", "done");
+		}
+
+	},
+
+	convert_preferences: function() {
+		const conversion = {
+			only_relative: {show_absolute: false},
+			only_inbox: {inbox: true, posts: false},
+			do_reblogs: {op: true, reblogs: true},
+			only_original: {reblogs: false}
+		};
```

Comment:
What about

```js
const conversion = [
    ['only_relative', {show_absolute: false}],
    ['only_inbox', {inbox: true, posts: false}],
    ['do_reblogs', {op: true, reblogs: true}],
    ['only_original', {reblogs: false}]
];
```

?

instead of doing this whole order thing